src/potree.ts

```diff
@@ -225,7 +237,7 @@ export class Potree implements IPotree {
         projectionFactor = halfHeight / (slope * distance);
       } else {
         const orthographic = camera as OrthographicCamera;
-        projectionFactor = radius / orthographic.top;
+        projectionFactor = 2 * halfHeight / (orthographic.top - orthographic.bottom)
```

Comment:
Could you please make a separate PR for this?

Reply:
Sure :) https://github.com/pnext/three-loader/pull/39